docs/flow/tutorials/adding-new-component.md

```diff
@@ -0,0 +1,20 @@
+---
+aliases:
+- /docs/agent/latest/flow/tutorials/adding-new-component
+title: Adding a New Component
```

Comment:
I'd like to suggest that we use these tutorials to teach Prometheus concepts through Flow, and to pick names that are focused more on the Prometheus aspect than the Flow aspect.

The content here is fine (it does teach Prometheus concepts), but the title can be reworked to something like "Mutating Prometheus Metrics"